x-pack/plugins/observability/public/pages/slos/helpers/sort_slos.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { SLO } from '../../../typings';
+import { SortType } from '../components/slo_list_search_filter_sort_bar';
+import { getSloDifference } from './get_slo_difference';
+
+export function sortSlos(sort: SortType | undefined) {
```

Comment:
🍰 nit: use a switch case and assertNever(sort) at the end of the function. So when we add new SortType, typescript will give us a nice error about the forgotten new case.

Reply:
I can't get this to work in this function; perhaps we can have a look together